### PR TITLE
New Story: replace form_with with simple_form for

### LINKED ISSUE #8: New Story: Display mythology names instead of ids
In the dropdown on the 'New Story' page, display the mythology names (including 'Unspecified') in the dropdown instead of the id numbers.

It would probably be nice to install and use the 'simple forms' gem:
https://github.com/heartcombo/simple_form

### PR DESCRIPTION
**Pull request by acelanoit**
## Problem

New Story: replaced the form_with with a simple_form_for.
This fully resolves #8

## Solution

Added gem 'simple_form' in Gemfile.
Updated  code in new.html.erb accordingly.

## Details

Also added one line of code to stories_controller to set the instance variable @mythologies again to ensure the form can be re-rendered properly if submit fails:
@mythologies = Mythology.all

### CODE REVIEW

**Review by mattlindsey — CHANGES_REQUESTED** [review]
Thanks. Works good. 
You can remove the file config/initializers/simple_form_bootstrap.rb since we are using Tailwind for styling, not Bootstrap.  Maybe see if adding this gem is good: https://github.com/tarellel/simple_form-tailwind.

Also, run `bundle exec rubocop` to look for any corrections, then `bundle exec rubocop -A` to correct them.  You can add the simple_form.rb file to the `Exclude:` section of rubocop.yml if you want to leave it as-is.

File: app/views/stories/new.html.erb [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
-    <%= form.text_field :title %>
-    <%= form.text_area :body, placeholder: 'AI will generate this', style: 'display:none' %>
+    <%= f.input :mythology_id, collection: @mythologies, label_method: :name, value_method: :id,
+    label_html: { style: 'margin-right: 15px;' },
+    input_html: { style: 'margin-bottom: 15px', class: 'rounded' } %>
+    <%= f.input :title, label_html: { style: 'margin-right: 49px;' },
+    input_html: { class: 'rounded' } %>
+    <%= f.hidden_field :body, input_placeholder: 'AI will generate this' %>
+    <div style="margin-top: 20px;">
+      <%= f.submit class: 'border font-bold py-2 px-4 rounded', style: 'display: block;' %>
```
Comment (mattlindsey) [inline app/views/stories/new.html.erb:21]: I think you can just add the 'block' class, but this is probably ok too.

**Comment by acelanoit** [comment]
Morning! I've made some changes based on your feedback, it looks much cleaner now, only left some inline style in to ensure the 2 input fields are perfectly aligned. Hope it's fine now (:

**Comment by mattlindsey** [comment]
Hi! Looks good. You just need to run `rubocop` and `rubocop -A` (see above) and add simple_form-tailwind gem (if you want).

**Comment by mattlindsey** [comment]
Actually `rubocop` is good, so merging now! Thanks

**Pull request merged**